src/components/Button.jsx

```diff
@@ -1,5 +1,19 @@
+import React from "react";
+import "tailwindcss/tailwind.css";
+
 const Button = () => {
-  return <div>Button</div>;
+  return (
+    <button
+      className="bg-rishi-white border border-rishi-orange text-rishi-black font-urbanist py-2 px-4 rounded"
+      style={{
+        width: "238px",
```

Comment:
don't use inline css, add all styles within className

Reply:
For sure, just deleted and pushed to my branch